CHANGELOG.txt

```diff
@@ -79,7 +85,7 @@ Enhancements:
 -------------
 
 * add --use-channeldata argument to conda render/build.
-* Extract the part in the skeletons pypi responsible to get the package metadata to a free function. 
+* Extract the part in the skeletons pypi responsible to get the package metadata to a free function.
```

Comment:
Could you please revert this file?
You can add the description and you changelog here
https://github.com/conda/conda-build/blob/master/news/

and you can also use our template
https://github.com/conda/conda-build/blob/master/news/TEMPLATE

:)

Reply:
Just add a new ``rst`` file on that folder using our template and it should be fine

Reply:
Will do that :)

Reply:
Does it look okay now?

Reply:
The file is still here =p

Reply:
Could you please revert it? After that, I think that should be good to go

Reply:
Oh, I thought I had reverted it. Tried it again. :)